test/e2e/rollout_test.go

```diff
@@ -27,35 +28,44 @@ import (
 	"go.goms.io/fleet/test/utils/controller"
 )
 
+const (
+	randomImageName = "random-image-name"
+	deploymentKind  = "Deployment"
+	daemonSetKind   = "DaemonSet"
+	statefulSetKind = "StatefulSet"
+	configMapKind   = "ConfigMap"
+	namespaceKind   = "Namespace"
```

Comment:
we have multiple definitions of those, is there any reason to define them again?

Reply:
Checked the repo, we don't have these consts defined in a common place so moved to pkg/utils